crates/core_arch/src/x86_64/xsave.rs

```diff
@@ -105,7 +105,7 @@ pub unsafe fn _xsaves64(mem_addr: *mut u8, save_mask: u64) {
     xsaves64(mem_addr, (save_mask >> 32) as u32, save_mask as u32);
 }
 
-/// Perform a full or partial restore of the enabled processor states using the
+/// Performs a full or partial restore of the enabled processor states using the
```

Comment:
Below this PR changes "Get" to "Gets", why go the other way around here?

Reply:
Wait, is this the other way round...? I thought I changed imperative to indicative everywhere. Maybe you could point out the other case, sorry.

Reply:
See https://github.com/rust-lang-nursery/stdsimd/pull/688/files#diff-e6a419cbbe65f76d6e34e2cff40c3d6fR93

Reply:
Oh. Well these are within bodies. The convention seems to be to use the indicative mood for function/method comments, but the imperative mood for (ordinary) comments within bodies.